src/lemlib/movements/purepursuit.cpp

```diff
@@ -87,7 +88,7 @@ std::pair<int, int> PurePursuit::update(Pose pose) {
     // find the closest waypoint on the path
     Waypoint closest = closestWaypoint(path, pose);
     // if the robot is at the end of the path, then stop
-    if (closest.speed == 0) {
+    if (closest.speed == 0_rpm) {
```

Comment:
```suggestion
    if (closest.speed == 0_inps) {
```
Change to correct unit